template/asyncapi.js

```diff
@@ -4,20 +4,22 @@ import { Header, Link, ListItem } from "../components/common";
 import { Info, TermsOfService } from "../components/Info";
 import { Servers } from "../components/Servers";
 import { Channels } from "../components/Channels";
+import { FrontMatter } from "../components/FrontMatter";
 
 export default function({ asyncapi, params }) {
   return (
     <File name={params.outFilename || 'asyncapi.md'}>
+      {params.frontMatter && <FrontMatter asyncapi={asyncapi} params={params} />}
       <Info asyncapi={asyncapi} params={params} />
-      <TableOfContents asyncapi={asyncapi} />
+      {params.toc !== 'false' && <TableOfContents asyncapi={asyncapi} params={params}/>}
       <TermsOfService asyncapi={asyncapi} />
       <Servers asyncapi={asyncapi} />
       <Channels asyncapi={asyncapi} />
     </File>
   );
 }
 
-function TableOfContents({ asyncapi }) {
+function TableOfContents({ asyncapi, params }) {
```

Comment:
Only remove `params={params}` from `TableOfContents` and `params` prop from this same component.

Reply:
yup, not needed anymore since `slate` param is out